### ISSUE TITLE
Active Learning Challenge Proceedings: Supplemental material not linked from proceedings page

### ISSUE THREAD

**Issue by mfeurer** [mlresearch/v16#1]
Hey,
I just found that the supplementary material (https://github.com/mlresearch/v16/tree/gh-pages/supplemental) is not linked from the pmlr page (http://proceedings.mlr.press/v16/).  It appears that they were linked before, because on google scholar I can find dead links pointing to the challenge report (see https://scholar.google.de/scholar?cluster=777890346025818308&hl=de&as_sdt=0,5&as_vis=1)

**Comment by lawrennd** [comment]
Thanks Matthias, I'm not sure how exactly to unpick this as I don't recall how they were linked before or what the supplementary contained. Can you clarify? Thanks!

**Comment by mfeurer** [comment]
Hi Neil,
Let me do my best to help here. First of all, I don't know whether they have been linked before. I was looking for the report on how the data for the challenge was derived, which was mentioned here http://proceedings.mlr.press/v16/guyon11a/guyon11a.pdf . Note that the link in the paper is dead. I then googled for the paper and found a link to the file on github in google scholar. 

This particular example contains a dataset description, a challenge description and two html files. I don't know whether they have been reviewed or not, and how they should be linked. I just thought that they should be linked as they provide valuable information.

Maybe we should try to reach out to Isabelle Guyon to solve this case?

I hope this helps a bit,
Matthias

**Comment by lawrennd** [comment]
Good idea, do you have her contact? I'm not sure if she has a GitHub we can @.

**Comment by mfeurer** [comment]
@madclam could you please help us here? We (Neil Lawrence and me) are discussing whether and how to link the supplementary material from your active learning challenge on the new proceedings of machine learning research page.

Our main question is whether you know whether the supplementary material has been linked before, and if yes, how it was linked.

**Comment by lawrennd** [comment]
The dataset and software reports now sit beside `guyon11a` as `{id}-supp` extras, matching the current proceedings format.

After GitHub Pages rebuilds they should appear next to *Results of the Active Learning Challenge* on https://proceedings.mlr.press/v16/:

- Dataset report: http://proceedings.mlr.press/v16/guyon11a/guyon11a-supp.pdf
- Software report: http://proceedings.mlr.press/v16/guyon11a/guyon11a-software.pdf

The old `supplemental/` HTML indexes are leftover JMLR W&CP wrappers and are not linked. Closing this issue.
